tests/bwc/test_recovery.py

```diff
@@ -0,0 +1,491 @@
+import time
+import unittest
+
+from crate.client import connect
+import random
+from random import sample
+
+from crate.qa.tests import NodeProvider, insert_data
+from bwc.test_rolling_upgrade import UpgradePath
```

Comment:
```suggestion
from .test_rolling_upgrade import UpgradePath
```

I think otherwise this will cause import errors depending on how the tests are launched.
Or even better: Move `UpgradePath` to `crate.qa.tests`